pkg/operator/ceph/cluster/crash/crash.go

```diff
@@ -182,6 +236,34 @@ func getCrashDaemonContainer(cephCluster cephv1.CephCluster, cephVersion version
 	return container
 }
 
+func getCrashPruneContainer(cephCluster cephv1.CephCluster, cephVersion version.CephVersion) corev1.Container {
+	cephImage := cephCluster.Spec.CephVersion.Image
+	envVars := append(controller.DaemonEnvVars(cephImage), generateCrashEnvVar())
+	dataPathMap := config.NewDatalessDaemonDataPathMap(cephCluster.GetNamespace(), cephCluster.Spec.DataDirHostPath)
+	volumeMounts := controller.DaemonVolumeMounts(dataPathMap, "")
+	volumeMounts = append(volumeMounts, keyring.VolumeMount().CrashCollector())
+
+	container := corev1.Container{
+		Name: "ceph-crash-pruner",
+		Command: []string{
+			"ceph",
+			"-n client.crash",
```

Comment:
```suggestion
			"-n",
			"client.crash",
```

Make `client.crash` a constant coming from `pkg/operator/ceph/cluster/crash/keyring.go`.